### DISCUSSION TITLE
Import to existing infrastructure

### DISCUSSION THREAD

**Discussion by ramazulay** [D10S0VSkY-OSS/Stack-Lifecycle-Deployment#63] [General]
Hi @D10S0VSkY-OSS,
Can I import existing infra to the new deploy in SLD?
I mean import the state file and continue from this state.(I already have a infra prod)

Thanks.

**Comment by D10S0VSkY-OSS** [comment]
Hi @ramazulay ,
importing resources is on the roadmap but for now it has not been addressed
Regards
